R/datasets.R

```diff
@@ -598,3 +598,23 @@
 #' \item{region}{Region Where Sale Took Place}
 #' }
 "avocados"
+
+
+#' House Prices and Properties in Saratoga, New York
+#' 
+#' Random sample of 1057 houses taken from full Saratoga Housing Data (De Veaux)
+#' @format A data frame with 1057 observations on the following 8 variables
+#' \describe{
+#' \item{Price}{Price (US dollars)} 
```

Comment:
@alejanmg @egordonhalpern note that after the names are cleaned using the updated code in `data-raw`, all variables will be in snake_case. So `Price` will now be `price`. The documentation here will also need to be updated.